### PR TITLE
Adds covid data subproject

### PR DESCRIPTION
**Pull request by AdrianKonca**
Adds two types of waves detection from data from ourworldindata.com
One based on hospitalized cases - which is more precise but not many country offer it. 
Another one based on active cases count - which is less precise but could still be useful gameplay wise.

### CODE REVIEW

File: Python/fetch.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+def hello_world():
+    print("hello")
```
Comment (mm44419) [inline Python/fetch.py]: Doesn't like it. To delete.

**Review by mm44419 — APPROVED** [review]
Very good Adrian, keep going

File: Python/fetch.py (hunk 1 above)
Comment (SebastianRatanczuk) [reply Python/fetch.py]: Why?

**Review by SebastianRatanczuk — APPROVED** [review]
Nice

**Comment by Krystian1222** [comment]
Nice.

File: Python/fetch.py (hunk 1 above)
Comment (Krystian1222) [reply Python/fetch.py]: Nice.

**Review by Krystian1222 — COMMENTED** [review]
Nice.

**Pull request merged**